apps/src/templates/ChallengeDialog.story.jsx

```diff
@@ -3,158 +3,135 @@ import CodeWritten from './feedback/CodeWritten';
-          >
-            <CodeWritten numLinesWritten={9} useChallengeStyles>
-              <GeneratedCode
-                message="Here's your code:"
-                code="console.log('F is friends who do stuff together!');"
-              />
-            </CodeWritten>
-          </ChallengeDialog>
-        </div>
-      )
-    },
-    {
-      name: 'Passed Dialog with a bird',
-      description: 'Too many blocks',
-      story: () => (
-        <div style={wrapperStyle}>
-          <ChallengeDialog
-            hideBackdrop
-            avatar="/blockly/media/skins/birds/win_avatar.png"
-            title="You did it!"
-            primaryButtonLabel="Continue"
-            cancelButtonLabel="Try again"
-            showPuzzleRatingButtons
-            text="However, you could've done it with only N blocks. Can you make your program even better?"
-          >
-            <CodeWritten numLinesWritten={9} useChallengeStyles>
-              <GeneratedCode
-                message="Here's your code:"
-                code="console.log('U is for you and me!');"
-              />
-            </CodeWritten>
-          </ChallengeDialog>
-        </div>
-      )
-    },
-    {
-      name: 'Perfect Dialog',
-      description: 'Perfect completion',
-      story: () => (
-        <div style={wrapperStyle}>
-          <ChallengeDialog
-            hideBackdrop
-            avatar="/blockly/media/skins/harvester/win_avatar.png"
-            complete
-            title="Challenge Complete!"
-            primaryButtonLabel="Continue"
-            cancelButtonLabel="Replay"
-            showPuzzleRatingButtons
-          >
-            <CodeWritten numLinesWritten={9} useChallengeStyles>
-              <GeneratedCode
-                message="Here's your code:"
-                code="console.log('N is for anywhere and anytime at all');"
-              />
-            </CodeWritten>
-          </ChallengeDialog>
-        </div>
-      )
-    },
-    {
-      name: 'Perfect Dialog without puzzle ratings',
-      description: 'Perfect completion',
-      story: () => (
-        <div style={wrapperStyle}>
-          <ChallengeDialog
-            hideBackdrop
-            avatar="/blockly/media/skins/studio/win_avatar.png"
-            complete
-            title="Challenge Complete!"
-            primaryButtonLabel="Continue"
-            cancelButtonLabel="Replay"
-          >
-            <CodeWritten numLinesWritten={9} useChallengeStyles>
-              <GeneratedCode
-                message="Here's your code:"
-                code="console.log('N is for anywhere and anytime at all');"
-              />
-            </CodeWritten>
-          </ChallengeDialog>
-        </div>
-      )
-    }
-  ]);
+// TEMPLATE
+
+const Template = args => (
+  <div style={wrapperStyle}>
+    <ChallengeDialog {...args} />
+  </div>
+);
+
+// STORIES
+
+export const Starting_Dialog = Template.bind({});
+Starting_Dialog.args = {
+  hideBackdrop: true,
```

Comment:
Maybe out of scope, but could we dedupe some of these props? I think either of the approaches discussed here would work:

https://github.com/code-dot-org/code-dot-org/pull/49129#discussion_r1022236463

Reply:
Good call, I'll pull the common ones in